clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/LongAdderRateGauge.java

```diff
@@ -24,4 +24,8 @@ public double measure(MetricConfig config, long now) {
     this.lastMeasurementTime = now;
     return (double) this.adder.sumThenReset() / elapsedTimeInSeconds;
   }
+
+  public double getRate(MetricConfig config) {
```

Comment:
I was thinking this static `METRIC_CONFIG` would become part of LARG since we don't even use it anyway. WDYT?